content/docs/guides/opentelemetry.md

```diff
@@ -67,6 +67,8 @@ otlp:
     - ...
 ```
 
+NOTE: This feature is not available in Prometheus 1.x and 2.x, consider upgrading to 3.x or include resource attributes at query time
```

Comment:
It looks as if this feature is to be included in [v2.55.0](https://github.com/prometheus/prometheus/blob/v2.55.0-rc.0/CHANGELOG.md#2550-rc0--2024-09-20).